recipes/openssl/1.x.x/conandata.yml

```diff
@@ -17,6 +17,12 @@ sources:
       "https://www.openssl.org/source/openssl-1.0.2u.tar.gz",
       "https://www.openssl.org/source/old/openssl-1.0.2u.tar.gz"
     ]
+  1.1.0g:
```

Comment:
OpenSSL patch versions are API and ABI compatible. There are no good reasons to use old patch versions with known vulnerabilities.

Beyond that, the entire 1.1.0 version series is end-of-life.

Old patch versions were removed on purpose when the OpenSSL recipe got transferred from conan-community to conan-center-index

At the very minimum, you should use 1.1.0l

(Reposting as a review, as PR comments are often overlooked)

Reply:
@CfirTsabari If you use OpenSSL for some internal application, which is security-wise irrelevant, I would suggest adding this version to your internal Conan remote

In any other case, using an outdated OpenSSL version for security-relevant cases with known vulnerabilities would be just irresponsible and I would like to avoid making it easier to use those versions by adding them to the CCI.

Even 1.1.0l is end-of-life and a security risk, but it has some vulnerabilities less than 1.1.0g and is a 100% drop-in replacement for 1.1.0g